wandb/sdk/wandb_run.py

```diff
@@ -44,7 +44,7 @@
 from wandb import errors
 from wandb import trigger
 from wandb._globals import _datatypes_set_callback
-from wandb.apis import internal, public
+from wandb.apis import internal, InternalApi, public
```

Comment:
You technically don't need this import. You should be able to do `internal.Api(...)` to init an internal API object.